lib/game.js

```diff
@@ -0,0 +1,101 @@
+function ToDoElement(value){
+  this.element = document.createElement("div");
+  const txtElement = document.createTextNode(value);
+  this.checkBox = document.createElement("input");
+  this.checkBox.setAttribute("type", "checkBox");
+  this.label = document.createElement("label");
+  this.label.appendChild(txtElement);
+  this.element.appendChild(this.checkBox);
+  this.element.appendChild(this.label);
+
+  this.checkBox.addEventListener("click", event=>{
```

Comment:
Nie potrzebujesz listenera zeby to ostylowac. Mozna to zrobic z css.
[checkbox](https://css-tricks.com/almanac/selectors/c/checked/)